File: src/main/java/com/telegram/folobot/domain/FoloVarEntity.java
package com.telegram.folobot.domain;

import lombok.*;
import lombok.experimental.Accessors;
import org.hibernate.Hibernate;

import javax.persistence.EmbeddedId;
import javax.persistence.Entity;
import javax.persistence.Table;
import java.util.Objects;

@Entity
@Getter
@Setter
@Accessors(chain = true)
@NoArgsConstructor
@RequiredArgsConstructor
@Table(name = "folo_var")
public class FoloVarEntity {
    @EmbeddedId
    @NonNull
    private FoloVarId id;
    @NonNull
    private String value;

    @Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (o == null || Hibernate.getClass(this) != Hibernate.getClass(o)) return false;
        FoloVarEntity foloVarEntity = (FoloVarEntity) o;
        return id != null && Objects.equals(id, foloVarEntity.id);
    }

    @Override
    public int hashCode() {
        return Objects.hash(id);
    }
}

File: src/main/kotlin/com/telegram/folobot/domain/FoloVarEntity.kt
package com.telegram.folobot.domain

import javax.persistence.Column
import javax.persistence.EmbeddedId
import javax.persistence.Entity
import javax.persistence.Table

@Entity
@Table(name = "folo_var")
class FoloVarEntity(
    @EmbeddedId
    @Column(nullable = false)
    var id: FoloVarId,
    @Column(nullable = false)
    var value: String
) {
}